src/Symfony/Component/Workflow/Registry.php

```diff
@@ -17,8 +17,6 @@
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
- *
- * @internal since Symfony 6.2. Inject the workflow where you need it.
```

Comment:
perhaps move it to `addWorkflow` instead

Reply:
addWorkflow is not internal. It is the only way to make an instance of the registry useful.

Reply:
excusi, there's no ctor -.-